### DISCUSSION TITLE
How to customize login page

### DISCUSSION THREAD

**Discussion by webmclink** [filamentphp/filament#9280] [Help]
How to customize the login page from filament 3?

**Comment by sinnbeck** [comment]
You should be able to return a view from the `->login()` method on the panel
https://filamentphp.com/docs/3.x/panels/users#authentication-features

```php
$panel
->login(fn() => view('login'))

**Reply by VincvanAs** [reply]
Please use this section as reference https://filamentphp.com/docs/3.x/panels/users#customizing-the-authentication-features

At the bottom there are other classes defind that you could extend to customize certain sections

**Comment by webmclink** [comment]
is there a way for me to publish the view and edit the login form? I'd like to change the logo and background.

**Reply by leandrocfe** [reply]
You can, but I wouldn't publish views. You could use this section to customize the logo and edit the login page via css.

https://filamentphp.com/docs/3.x/panels/themes/
